configs/config/pretrain/byol/byol_1gpu.yaml

```diff
@@ -0,0 +1,121 @@
+# @package _global_
+config:
+  MAX_ITER: 312785 # int(1000. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
```

Comment:
I don't believe: MAX_ITER and WARMUP_ITERS are valid config options. 

I see max_iter under SVM, but that is not relevant. WARMUP_ITERS looks deprecated. Both of these concepts should be taken care of in `num_epochs` and `config.OPTIMIZER.lr`.